packages/cli/src/lib/setups/i18n/replacers.test.ts

```diff
@@ -276,4 +283,689 @@ describe('i18n replacers', () => {
       `);
     });
   });
+
+  it('replace cart path for i18n subfolder strategy', async () => {
+    await inTemporaryDirectory(async (tmpDir) => {
+      const skeletonDir = getSkeletonSourceDir();
+      await copyFile(joinPath(skeletonDir, cartTsx), joinPath(tmpDir, cartTsx));
+      await copyFile(
+        joinPath(skeletonDir, addToCartButtonTsx),
+        joinPath(tmpDir, addToCartButtonTsx),
+      );
+      await copyFile(joinPath(skeletonDir, rootTsx), joinPath(tmpDir, rootTsx));
+
+      await replaceI18nCartPath({rootDirectory: tmpDir}, {});
+
+      const newRootFile = await readFile(joinPath(tmpDir, rootTsx));
+      expect(() => checkTypes(newRootFile)).not.toThrow();
+
+      expect(newRootFile).toMatchInlineSnapshot(`
```

Comment:
Since the changes in these files are rather small, maybe we should assert changes using regexps instead of inline snapshots? Otherwise we'd need to be updating these snapshots every time we update `root.tsx` etc.
But not a blocker 👍